openshift_tools/monitoring/pminfo.py

```diff
@@ -115,6 +118,39 @@ def connect(self):
         """ Establish a PMAPI context to archive, host or local, via args """
         self.context = pmapi.pmContext()
 
+def calculate_percent_cpu(start, end, interval):
+    ''' Calculate the percent of cpu utilization from sample1 and sample2.
+        Requires:
+          start
+          end
+          number of cpus
+          interval
+    '''
+    num_cpu = multiprocessing.cpu_count()
+    jiffy = os.sysconf(os.sysconf_names['SC_CLK_TCK'])
+    return ((end - start) * 100 / (float(jiffy) * interval)) / (num_cpu * 10)
+
+def get_sampled_data(metrics, interval, count=1):
+    '''Recieve a list of metrics and a time interval.
+       sample the metrics, sleep, sample, and then calculate the values.
+    '''
+    if not metrics:
+        return {}
+
+    sampled_results = []
+    for i in range(count):
+        sampled_results.append(get_metrics(metrics))
+        if i == count-1:
+            break
+        time.sleep(interval)
```

Comment:
ok, I'm torn on this, so I definitely want your opinion.

Should interval be the time between checks, or the total time they want sampled.

In other words, if I say I want an interval of 60 seconds and a count of 10, should that take 1 minute or 10 minutes?

I think I'm leaning towards the interval being the entire time, which would make this line simply:
```
time.sleep(interval / count)
```

That seems to make the most sense.

Thoughts?